grafana/PostgreSQLDetails.json

```diff
@@ -846,28 +845,74 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(ccp_stat_database_deadlocks{job=\"[[jobname]]\"}[5m]))",
+          "expr": "sum(ccp_locks_count{job=\"[[jobname]]\",mode=\"accessexclusivelock\"})",
           "format": "time_series",
           "intervalFactor": 2,
-          "legendFormat": "Conflicts",
-          "metric": "pg_stat_database_conflicts",
+          "legendFormat": "accessexclusive",
           "refId": "A",
           "step": 240
         },
         {
-          "expr": "sum(rate(ccp_stat_database_conflicts{job=\"[[jobname]]\"}[5m]))",
+          "expr": "sum(ccp_locks_count{job=\"[[jobname]]\",mode=\"accesssharelock\"})",
           "format": "time_series",
           "intervalFactor": 2,
-          "legendFormat": "DeadLocks",
-          "metric": "pg_stat_database_deadlocks",
+          "legendFormat": "accessshare",
           "refId": "B",
           "step": 240
+        },
```

Comment:
may it is git or github but at this level change looks questionable, showing as lot of additions.

Reply:
VSCode auto formats json on save.